tox.ini

```diff
@@ -0,0 +1,118 @@
+# Managing a Project's Virtualenvs with tox
+# A comprehensive beginner's introduction to tox.
+# Tox is a great tool for standardising and automating any development task
+# that benefits from being run in an isolated virtualenv:
+# https://www.seanh.cc/2018/09/01/tox-tutorial/
+# To run the tests for all supported and installed python versions, run:
+# pip3 install 'py>=1.11.0' tox; tox
+
+# Installation of additional python versions:
+# - On Fedora 37, `sudo dnf install tox` installs all Python versions, even 3.12a6.
+# - On Ubuntu, the deadsnakes/ppa is broken(except for 3.12), so conda or pyenv has to be used:
+#   sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev
+#                           libreadline-dev libsqlite3-dev xz-utils libffi-dev liblzma-dev
+#    curl https://pyenv.run | bash # Then install the displayed commands in your .bashrc.
+#    pyenv install 3.{6,7,8,9} && pyenv local 3.{6,7,8,9} # builds and add them to PATH.
+#    https://brandonrozek.com/blog/pyenvtox/; https://realpython.com/intro-to-pyenv/
+#
+# - Note: virtualenv 20.22 broke support for the py27 venv with tox. (at least in some setups)
+#   As a workaround, downgrade it to 20.21 if that happens: pip3 install -U 'virtualenv<20.22'
+# - For testing on newer Ubuntu hosts which have python2-dev, but not pip2, install pip2 this way:
+#   curl https://bootstrap.pypa.io/pip/2.7/get-pip.py --output get-pip.py;sudo python2 get-pip.py
+
+[tox]
+skipsdist = true
```

Comment:
This should be only set to true for applications: https://tox.wiki/en/4.5.1/config.html#skipsdist
```suggestion
```